plugins/woocommerce/client/admin/client/settings-payments/settings-payments-offline.scss

```diff
@@ -1,9 +1,6 @@
+@import 'settings-payments-main.scss';
+
 .settings-payments-offline__container {
-	h1 {
-		color: #fff;
-	}
-	background: #333;
-	text-align: center;
-	padding: 50px 0;
-	width: 100%;
+	// Temporarily copy styles while things are under construction.
+  @extend .settings-payments-main__container;
```

Comment:
This is currently re-using the same CSS using extend since there's WIP, we'll eventually need to refactor to combine stylings in follow-ups